src/components/BaseWizard/hooks/useWizardProps.js

```diff
@@ -0,0 +1,86 @@
+import { computed, ref } from '@vue/composition-api';
+import getProgressStepperOptions from '../utils/getProgressStepperOptions';
+
+const props = ref();
+
+const isLoading = computed(() => props.value.isLoading);
+
+const isFirstStep = computed(() => props.value.isFirstStep);
+const isLastStep = computed(() => props.value.isLastStep);
+
+const fullScreen = computed(() => props.value.fullScreen);
+
+const showFooter = computed(() => props.value.showFooter);
+
+const showProgress = computed(() => props.value.showProgress);
+
+const title = computed(() => props.value.title);
+
+const isComplete = computed(() => props.value.isComplete);
+
+const compact = computed(() => props.value.compact);
+
+const currentLayout = computed(() => props.value.layout);
+const mainContent = ref(null);
+const currentStep = computed(() => props.value.currentStep);
+const hasLabels = ref(false);
+
+/** BUTTON LOGIC */
+const buttonSize = computed(() => (props.value.compact ? 'medium' : 'large'));
+
+/** PROGRESS BAR LOGIC */
+const totalProgressSteps = computed(() => props.value.steps.length);
+
+const currentStepIndex = computed(() =>
+  props.value.steps.indexOf(currentStep.value)
+);
+
+const progressOptions = computed(() =>
+  getProgressStepperOptions(props.value.steps, hasLabels)
+);
+
+const progressStepperOptions = computed(
+  () => progressOptions.value.stepperOptions
+);
+
+const currentProgressStep = computed(
+  () => `${props.value.steps.indexOf(currentStep.value)}` // Requires a String value
+);
+
+const percentComplete = computed(() =>
+  props.value.isComplete
+    ? 100
+    : (currentStepIndex.value / props.value.steps.length) * 100
+);
```

Comment:
This could be: 
```suggestion
const percentComplete = computed(() =>
  isComplete.value
    ? 100
    : (currentStepIndex.value / props.value.steps.length) * 100
);
```

Right?